tools/osbuild-mpp

```diff
@@ -1406,28 +1409,40 @@ class ManifestFileV2(ManifestFile):
         def embed_data(ip, mpp):
             uid = mpp["id"]
             path = mpp.get("path")
+            url = mpp.get("url")
             text = mpp.get("text")
 
-            if path and text:
-                raise ValueError(f"Cannot specify both 'path' and 'text' for '{uid}'")
+            input_count = (1 if text else 0) + (1 if path else 0) + (1 if url else 0)
```

Comment:
How about:

```suggestion
            input_count = bool(text) + bool(path) + bool(url)
```

Reply:
I didn't know this one. (You learn something new every day :-) ). Thanks